website/content/docs/agent/template.mdx

```diff
@@ -120,12 +119,39 @@ template {
   source      = "/tmp/agent/template.ctmpl"
   destination = "/tmp/agent/render.txt"
 }
+
+template_retry {
+  enabled = true
+  attempts = 5
+  backoff = "100ms"
+  max_backoff = "400ms"
+}
 ```
 
 If you only want to use the Vault agent to render one or more templates and do
 not need to sink the acquired credentials, you can omit the `sink` stanza from
 the `auto_auth` stanza in the agent configuration.
 
+The `template_retry` stanza controls the retry behavior when an error is returned
+from Vault. Vault Agent is highly fault tolerant, meaning it does not exit in the
+face of failure. Instead, it uses exponential back-off and retry functions
```

Comment:
Can/should we mention here that this stanza is specifically for the template server?